tests/e2e/workloads/app/test_jenkins_simulation.py

```diff
@@ -41,5 +42,28 @@ class TestJenkinsSimulation(ManageTest):
     def test_git_clone(self, pod):
         """
         git clones a large repository
+        Added test coverage for BZ #2096395
         """
+        csi_cephfsplugin_pod_objs = res_pod.get_all_pods(
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            selector=["csi-cephfsplugin"],
+        )
+
+        relevant_pod_logs = None
+        func_calls = ["NodeStageVolume", "NodeGetVolumeStats"]
+        error_msg = "System has not been booted with systemd"
+        for p in csi_cephfsplugin_pod_objs:
+            pod_log = res_pod.get_pod_logs(
+                pod_name=p.name, container="csi-cephfsplugin"
+            )
+            if any(f_call in pod_log for f_call in func_calls):
+                relevant_pod_logs = pod_log
+                logger.info(f"Found logs on pod {p.name}")
```

Comment:
can we make it more clear on what function call log are found in the log message?

Reply:
Done